plugins/UFM_NDT_Plugin/ndt_file_creator.py

```diff
@@ -229,19 +226,19 @@ def main():
     #rack #,U height,#Fields:StartDevice,StartPort,StartDeviceLocation,EndDevice,EndPort,EndDeviceLocation,U height_1,LinkType,Speed,_2,Cable Length,_3,_4,_5,_6,_7,State,Domain
     #,,SwitchX -  Mellanox Technologies,Port 26,,r-ufm64 mlx5_0,Port 1,,,,,,,,,,,,Active,In-Scope
     header="rack #,U height,#Fields:StartDevice,StartPort,StartDeviceLocation,EndDevice,EndPort,EndDeviceLocation,U height_1,LinkType,Speed,_2,Cable Length,_3,_4,_5,_6,_7,State,Domain\n"
-    with open(OUTPUT_NDT_FILE_NAME, 'w') as topoconfig_file:
-        topoconfig_file.write(header)
+    with open(OUTPUT_NDT_FILE_NAME, 'w') as ndt_file:
+        ndt_file.write(header)
         for link in ibdiagnet_links:
             line = ",,%s,Port %s,,%s,Port %s,,,,,,,,,,,,Active,In-Scope\n" % (
                    link.start_dev, link.start_port, link.end_dev, link.end_port)
-            topoconfig_file.write(line)
+            ndt_file.write(line)
         for disconnected_port in links_list_disconnected:
             line = ",,%s,Port %s,,,,,,,,,,,,,,,Disabled,Disconnected\n" % (
                                         disconnected_port.get("node_name", ""),
                                         disconnected_port.get("node_port_number", ""))
-            topoconfig_file.write(line)
+            ndt_file.write(line)
 
-    print("NDT file generation completed. Could be found at %s" % OUTPUT_NDT_FILE_NAME)
+    print("NDT file generation completed. Generate file can be found at %s" % OUTPUT_NDT_FILE_NAME)
```

Comment:
```suggestion
    print("NDT file generation completed. Generated file can be found at %s" % OUTPUT_NDT_FILE_NAME)
```

Reply:
Eylon, This is what you requested to fix last time!

Reply:
yes, but there is a typo...need to fix.